Automattic/wp-calypso
# Introduce block-renderer to site assembler to improve performance

#### Proposed Changes

This PR is focusing on the block-renderer package and uses it to improve the performance of the site assembler. See the simple [README.md](https://github.com/Automattic/wp-calypso/blob/a58ad832102e59f4aa8b856bfb063390d7df3c6c/packages/block-renderer/README.md) to understand how to use the block-renderer package.

pbxlJb-39t-p2 describes the idea of the block-renderer, and pbxlJb-3gc-p2 does the performance analysis

Here is a follow-up PR to handle the scroll behavior: https://github.com/Automattic/wp-calypso/pull/71527

https://user-images.githubusercontent.com/13596067/209108222-d3114764-04b4-4d73-838f-9c5f35a75288.mov

#### Testing Instructions



* Apply D95663-code to your sandbox
* Go to `/setup?siteSlug=<your_site>&flags=pattern-assembler/client-side-render`
* Click the "Continue" button until you land on the Design Picker step
* Scroll to the bottom and select the Blank Canvas CTA
* When you're in the Pattern Assembler step
  * Verify every pattern in the pattern list looks good as before
  * Verify the large preview looks good after you select any header, sections, and footers

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/67076
